.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Request For Comments (RFC)
+    about: When having an idea on how to improve our processes, propose the idea so that the team can provide feedback.
+    url: https://nimble.link/android-templates-rfc
+
+  - name: Ask a question
+    about: When having a question about a pull request, issue, or problem, create a Q&A discussion to get help from the team.
+    url: https://github.com/nimblehq/android-templates/discussions/new?category=q-a
```

Comment:
@Tuubz Just wondering, why do you create shorten link for RFC only?

Reply:
It's nice to have a shortened link for QA 🙆‍♂️ In the future we don't need to update the shortened link if our discussion link update, I guess 💭

Reply:
So we have two choices here:

A. Shorten links for both
B. Don't shorten links for both

My vote goes for `B` to keep it simple 🙏

Reply:
@Tuubz let's go with it then 🙏